videos/tasks.py

```diff
@@ -216,7 +216,9 @@ def delete_s3_objects(
 def update_transcripts_for_video(video_id: int):
     """Update transcripts for a video"""
     video = Video.objects.get(id=video_id)
+    log.error("update_transcripts_for_video")
```

Comment:
Seems like we maybe have some stray debug logging here?

Reply:
removed